<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>1165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.267045, y1:0.221591, x2:1, y2:1, stop:0 rgba(157, 210, 216, 214), stop:1 rgba(106, 199, 229, 244))}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0.267045, y1:0.221591, x2:1, y2:1, stop:0 rgba(157, 210, 216, 214), stop:1 rgba(106, 199, 229, 244))}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox#groupBox{
font: 25pt &quot;华光行书_CNKI&quot;;

}
</string>
         </property>
         <property name="title">
          <string>选项</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="heartsound_Button">
            <property name="font">
             <font>
              <family>华光行书_CNKI</family>
              <pointsize>25</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#heartsound_Button{
font: 25pt &quot;华光行书_CNKI&quot;;
border-radius:20px;
background-color: #EBEFF8;}</string>
            </property>
            <property name="text">
             <string>心音信号分析</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="respiratory_Button">
            <property name="font">
             <font>
              <family>华光行书_CNKI</family>
              <pointsize>25</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#respiratory_Button{
font: 25pt &quot;华光行书_CNKI&quot;;
border-radius:20px;
background-color: #EBEFF8;}</string>
            </property>
            <property name="text">
             <string>呼吸音信号分析</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="chat_Button">
            <property name="font">
             <font>
              <family>华光行书_CNKI</family>
              <pointsize>25</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#chat_Button{
font: 25pt &quot;华光行书_CNKI&quot;;
border-radius:20px;
background-color: #EBEFF8;}</string>
            </property>
            <property name="text">
             <string>医患交流</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="notebook_Button">
            <property name="font">
             <font>
              <family>华光行书_CNKI</family>
              <pointsize>25</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#notebook_Button{
font: 25pt &quot;华光行书_CNKI&quot;;
border-radius:20px;
background-color: #EBEFF8;}</string>
            </property>
            <property name="text">
             <string>病症记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hospital_Button">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#hospital_Button{
font: 25pt &quot;华光行书_CNKI&quot;;
border-radius:20px;
background-color: #EBEFF8;}</string>
            </property>
            <property name="text">
             <string>医院信息查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="username">
            <property name="styleSheet">
             <string notr="true">font: 15pt &quot;幼圆&quot;;
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
